crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -262,6 +262,7 @@ def status
                 "pending" => 0,
                 "unready" => 0,
                 "building" => 0,
+                "crowbar-upgrade" => 0,
```

Comment:
This is port of https://github.com/crowbar/barclamp-crowbar/pull/1358.
We might need `upgrade` for other tasks, that's why we've chosen `crowbar-upgrade`